.github/workflows/website-deploy-release.yml

```diff
@@ -0,0 +1,56 @@
+name: "website-deploy-release"
```

Comment:
I believe you can combine the release and preview workflows by using `if` conditions on the build steps

Reply:
I thought about it, but did not want to 1) complicate the code in each workflow; 2) we always did it this way by separating the release workflow from the preview workflow (including doing it this way in Codefresh); 3) when the workflow runs, you can see in the GitHub UI what it is about